### ISSUE TITLE
[Crash report]  While reading from '/home/USER/.

### ISSUE THREAD

**Issue by benopotamus** [bottlesdevs/Bottles#2334]
tl;dr I fixed this error by deleting the game directory in my Documents folder.

---------------------------------------

I had some trouble with Overwatch 2 and I tried uninstalling the bottle but it didn't seem to work. The bottle was still in the list inside Bottles. The programs in the bottle were mostly removed, but there was one unnamed one (blank name) that just gave errors in a Wine dialog box. I then deleted the `/home/USER/.local/share/bottles/bottles/Overwatch-2` directory to see if that would help. 

This is the error I received when trying to install Battle.net (needed for Overwatch 2) again. After receiving this error, I  deleted `/users/ben/Documents/Overwatch/` and was then able to successfully install Battle.net.

-----------------------------------

This crash report was generated by Bottles.

**Details**
* **package**: Other
* **version**: 2022.8.28-brescia-2

**Log**
```python3
While reading from '/home/USER/.local/share/bottles/bottles/Overwatch-2/drive_c/users/ben/Documents/Overwatch/Settings/Settings_v0.ini' [line 11]: option 'movieexport.1' in section 'General.0' already exists  File "/usr/share/bottles/bottles/frontend/utils/threading.py", line 63, in __target
    result = self.task_func(*args, **kwargs)

  File "/usr/share/bottles/bottles/backend/managers/installer.py", line 481, in install
    if not self.__perform_steps(_config, steps):

  File "/usr/share/bottles/bottles/backend/managers/installer.py", line 196, in __perform_steps
    self.__step_update_config(config, st)

  File "/usr/share/bottles/bottles/backend/managers/installer.py", line 293, in __step_update_config
    _conf = ConfigManager(config_file=conf_path, config_type=conf_type)

  File "/usr/share/bottles/bottles/backend/managers/conf.py", line 15, in __init__
    self.config_dict = self.read()

  File "/usr/share/bottles/bottles/backend/managers/conf.py", line 33, in read
    config.read(self.config_file)

  File "/usr/lib64/python3.10/configparser.py", line 698, in read
    self._read(fp, filename)

  File "/usr/lib64/python3.10/configparser.py", line 1097, in _read
    raise DuplicateOptionError(sectname, optname,

```

**Comment by mirkobrombin** [comment]
2022.8.28-brescia-2 is too old, please upgrade
